tests/phpunit/tests/comment/metaCache.php

```diff
@@ -77,8 +150,6 @@ public function test_update_comment_meta_cache_true() {
 	 * @covers ::update_comment_meta
 	 */
 	public function test_update_comment_meta_cache_false() {
```

Comment:
For cleanliness of commit history, I think the changes below switching from wpdb to get_num_queries would be better done on the generic _unit test improvements for 6.3_ ticket, [WP#57841](https://core.trac.wordpress.org/ticket/57841).

Reply:
Noted. I will commit this before I commit the rest of the code.

Reply:
@peterwilsoncc Created a PR for use of `get_num_queries`. https://github.com/WordPress/wordpress-develop/pull/4438